travis/test_server.py

```diff
@@ -15,6 +16,11 @@
 from travis_helpers import success_msg, fail_msg
 
 
+if not sys.version_info > (3, 0):
+    reload(sys)
+    sys.setdefaultencoding('utf8')
```

Comment:
@moylop260 
What do you think about this solution ?

Reply:
What about use magic comment at top with the enconding?